search/serverless.yml

```diff
@@ -47,6 +48,7 @@ custom:
   burstComputeStage: ${env:BURST_COMPUTE_STAGE, self:provider.stage}
   tasksTable: ${cf:burst-compute-${self:custom.burstComputeStage}.TasksTable}
   parallelDispatchFunction: ${cf:burst-compute-${self:custom.burstComputeStage}.DispatchLambdaFunction}
+  toleratedPercentageFailure: ${env:TOLERATED_PERCENTAGE_FAILURE, 10}
```

Comment:
this is a new attribute used by the new burst compute that uses the Step Function - Map state to parallelize the work. A Map state can still complete successfully if the percentage of the failed jobs is less that this value here so if you think that a default of 10% is too high let me know.

Reply:
Do we regularly have this many failures? If so, we should aim to reduce that to zero. Ideally, there should never be any failures, it should just retry until it succeeds, even if that takes an hour.

Reply:
I will reduce it and increase the number of retries